.github/workflows/test.yml

```diff
@@ -93,8 +93,8 @@ jobs:
         uses: ./coverage-cop
         with:
           path: ./build/coverage.info
-          branch-min-coverage: 100
-          line-min-coverage: 99
+          branch-coverage-min: 70
```

Comment:
Why is 70 the right value here?

Reply:
No specific reason for choosing 70. Any value is works as long as the test job for the action uses different values for the line and branch coverages to ensure to make it visible in logs that the action is correctly using the input parameters.